sdk/core/core-asynciterator-polyfill/README.md

```diff
@@ -1,19 +1,21 @@
 # Azure Async Iterator Polyfill client library for JS
 
-This library acts to polyfill for Symbol.asyncIterator
+This library provides a polyfill for [Symbol.asyncIterator](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Symbol/asyncIterator)
+for platforms that do not have support for it by default.
 
 ## Getting started
 
-### Requirements
-- node.js version > 6.x
-- npm install -g typescript
-
 ### Installation
-- After cloning the repo, execute `rush install`
 
-## Key concepts
+If using this as part of another project in the [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js) repo,
```

Comment:
Should not use "simply", "just", etc. in technical documentation.

Reply:
fixed